charts/newrelic-infra-operator/templates/deployment.yaml

```diff
@@ -81,3 +81,4 @@ spec:
       affinity:
         {{- . | nindent 8 -}}
       {{- end }}
+      hostNetwork: {{ include "newrelic.common.hostNetwork.value" . }}
```

Comment:
`hostNetwork` could break the DNS.

Take a look at this: https://github.com/newrelic/helm-charts/pull/802/files#diff-394b2b36b0f27c9bf3f7555749adc69f9f92db9fc33020f4393d566efc10933aR43-R46

This issue is in more charts that I have to revisit yet because I did not properly review them 😅

Reply:
Thanks!

fixed on https://github.com/newrelic/newrelic-infra-operator/pull/166/commits/b15cb9bd0442d601a68b525a27b11525544c4020